safe/definitions/hazard_exposure_specifications.py

```diff
@@ -55,7 +63,15 @@
                 'model.'),
             # end notes provided by Hadi Ghasemi
         ]
-    }
+    },
+    {
+        'hazard': hazard_flood,
+        'exposure': exposure_road,
+        'analysis_question': {
+            'fr': u"Dans l'événement d'une inondation, quelle longueur de "
```

Comment:
I'm just asking the possibility here.
Does it have to be hardcoded?
Or can we use tr('Some unique statement about this case'),
then translate it (put the translated string) on transifex?